extensions/binding/org.eclipse.smarthome.binding.ntp/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ntp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ntp">
+		<label>Date</label>
+		<description>NTP Updated Date &amp; Time</description>

```

Comment:
The "Thing" describes the physical machine that is the source of the information. In this case, it is the server for which you define the hostname. It is definitely a server, not a client. And afaik, a server that provides current time&date through NTP is very commonly referred to as an "NTP server".